mqtt/mqtt-edgehub/src/auth/authorization/edgehub.rs

```diff
@@ -14,23 +14,32 @@ use mqtt_broker::{
 /// telemetry messages, etc...
 ///
 /// For non-iothub-specific primitives it delegates the request to an inner authorizer (`PolicyAuthorizer`).
-#[derive(Debug)]
```

Comment:
why remove?

Reply:
I guess with intermediate changes I doesn't want to be Debug. and we don't use it anywhere anyways